content/2019-09-23_PyConCZ_wrap_up_summary.md

```diff
@@ -25,81 +27,82 @@ If you’d be interested in joining our org team, don’t hesitate to write to u
 
 ### Tickets
 
-There were 457 registered attendees (or 498 including organizers, sponsors’ booths staff etc.) and 94 % of them have checked-in at the conference.
+There were 457 registered attendees (or 498 including organizers, sponsors’ booths staff etc.) and 94 % of them have checked in at the conference.
 
-We offered again a limited number of “Early bird” tickets. This year more than 6 weeks sooner than in 2018 and kept the “Regular” prices until 3 weeks before the event. Despite a larger venue, we sold out all tickets again, this time a few days after the “Late bird” period started.
+Like last year, we offered a limited number of “Early bird” tickets. This year, we did so more than 6 weeks sooner than in 2018 and kept the “Regular” prices until 3 weeks before the event. Despite a larger venue, we sold out all tickets again, this time a few days after the “Late bird” period would have started.
```

Comment:
> "this time a few days after the “Late bird” period would have started"

@encukou The late bird period *did* start before we sold out, so "would have" is incorrect I believe.

Reply:
```suggestion
Like last year, we offered a limited number of “Early bird” tickets. This year, we did so more than 6 weeks sooner than in 2018 and kept the “Regular” prices until 3 weeks before the event. Despite a larger venue, we sold out all tickets again, this time a few days after the “Late bird” period started.
```

Reply:
Juj, to jsem přečetl špatně. Díky!